google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsConstants.java

```diff
@@ -55,6 +55,28 @@ public class BuiltinMetricsConstants {
   static final AttributeKey<String> TRANSPORT_ZONE = AttributeKey.stringKey("transport_zone");
   static final AttributeKey<String> TRANSPORT_SUBZONE = AttributeKey.stringKey("transport_subzone");
 
+  // gRPC attribute keys
+  static final AttributeKey<String> GRPC_LB_BACKEND_SERVICE_KEY =
+      AttributeKey.stringKey("grpc_lb_backend_service");
+  static final AttributeKey<String> GRPC_DISCONNECT_ERROR_KEY =
+      AttributeKey.stringKey("grpc_disconnect_error");
+  static final AttributeKey<String> GRPC_LB_LOCALITY_KEY =
+      AttributeKey.stringKey("grpc_lb_locality");
```

Comment:
I think this should be 
```suggestion
      AttributeKey.stringKey("grpc.lb.locality");
```
?